Error: Cannot find module '@tanstack/start-client-core/package.json'

### Which project does this relate to?

Start

### Describe the bug

I just went through this "get started from scratch "tutorial: https://tanstack.com/start/latest/docs/framework/react/build-from-scratch

I did it as an app inside my pnpm workspaces monorepo. Once I finished copy-pasting the files, I ran `vite dev` and I immediately got this error:

```
> vite dev

failed to load config from /Users/me/my-repo/apps/consumer/vite.config.ts
error when starting dev server:
Error: Cannot find module '@tanstack/start-client-core/package.json'
Require stack:
- /Users/me/my-repo/node_modules/.pnpm/@tanstack+start-plugin-core@1.132.48_@tanstack+react-router@1.132.47_react-dom@19.2.0_r_46853f367f420c659285e6d8a9137776/node_modules/@tanstack/start-plugin-core/dist/esm/start-compiler-plugin/plugin.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1365:15)
    at Function.resolve (node:internal/modules/helpers:145:19)
    at resolvePackage (file:///Users/me/my-repo/node_modules/.pnpm/@tanstack+start-plugin-core@1.132.48_@tanstack+react-router@1.132.47_react-dom@19.2.0_r_46853f367f420c659285e6d8a9137776/node_modules/@tanstack/start-plugin-core/dist/esm/start-compiler-plugin/plugin.js:20:41)
    at resolveRuntimeFiles (file:///Users/me/my-repo/node_modules/.pnpm/@tanstack+start-plugin-core@1.132.48_@tanstack+react-router@1.132.47_react-dom@19.2.0_r_46853f367f420c659285e6d8a9137776/node_modules/@tanstack/start-plugin-core/dist/esm/start-compiler-plugin/plugin.js:15:19)
    at startCompilerPlugin (file:///Users/me/my-repo/node_modules/.pnpm/@tanstack+start-plugin-core@1.132.48_@tanstack+react-router@1.132.47_react-dom@19.2.0_r_46853f367f420c659285e6d8a9137776/node_modules/@tanstack/start-plugin-core/dist/esm/start-compiler-plugin/plugin.js:45:18)
    at TanStackStartVitePluginCore (file:///Users/me/my-repo/node_modules/.pnpm/@tanstack+start-plugin-core@1.132.48_@tanstack+react-router@1.132.47_react-dom@19.2.0_r_46853f367f420c659285e6d8a9137776/node_modules/@tanstack/start-plugin-core/dist/esm/plugin.js:253:5)
    at tanstackStart (file:///Users/me/my-repo/node_modules/.pnpm/@tanstack+react-start@1.132.48_react-dom@19.2.0_react@19.2.0__react@19.2.0_vite@7.1.9_@_dce01f992b65ffea061cf1f404a221e9/node_modules/@tanstack/react-start/dist/esm/plugin/vite.js:55:5)
    at file:///Users/me/my-repo/apps/consumer/node_modules/.vite-temp/vite.config.ts.timestamp-1759862193766-65a11ae97f1a5.mjs:12:5
    at ModuleJob.run (node:internal/modules/esm/module_job:343:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:647:26)
 ELIFECYCLE  Command failed with exit code 1.
```

### Your Example Website or App

working on it

### Steps to Reproduce the Bug or Issue

1. Follow the instructions in this tutorial: https://tanstack.com/start/latest/docs/framework/react/build-from-scratch. Copy all the files verbatim, except build it within a pnpm-workspaces monorepo.
2. Run `vite dev`.

### Expected behavior

I expected vite to start without failing to load any files or modules.

### Screenshots or Videos

_No response_

### Platform

- Router / Start Version: 1.132.47 / 1.132.48
- OS: macOS
- Browser: Irrelevant
- Browser Irrelevant
- Bundler: vite
- Bundler Version: 7.1.9


### Additional context

_No response_

--- a/packages/start-plugin-core/package.json
+++ b/packages/start-plugin-core/package.json
@@ -70,6 +70,7 @@
     "@tanstack/router-plugin": "workspace:*",
     "@tanstack/router-utils": "workspace:*",
     "@tanstack/server-functions-plugin": "workspace:*",
+    "@tanstack/start-client-core": "workspace:*",
     "@tanstack/start-server-core": "workspace:*",
     "babel-dead-code-elimination": "^1.0.9",
     "cheerio": "^1.0.0",

